en_pud-ud-test.conllu

```diff
@@ -11754,7 +11754,7 @@
 28	of	of	ADP	IN	_	29	case	29:case	Proper=True
 29	Panama	Panama	PROPN	NNP	Number=Sing	27	nmod	27:nmod:of	SpaceAfter=No
 30	)	)	PUNCT	-RRB-	_	20	punct	20:punct	_
-31	as	as	SCONJ	IN	_	37	mark	37:mark	_
+31	as	as	SCONJ	IN	ExtPos=SCONJ	37	mark	37:mark	_
```

Comment:
In this context, if "as opposed to" is a fixed expression, it functions like a preposition ("as+opposed+to the North Sea"), so ExtPos=ADP, 37:case, and "Sea" should be 11:obl:as_opposed_to not advcl.

BUT I'm not sure this is actually the grammaticized usage of "as opposed to". The sentence:

- From the 16th century, Europeans visiting the Caribbean region identified the "South Sea" (the Pacific Ocean, to the south of the isthmus of Panama) as opposed to the "North Sea" (the Caribbean Sea, to the north of the same isthmus).

Does this mean 'they identified the South Sea—they did not identify the North Sea'? Or is it meant as a definitional contrast: 'they singled out the South Sea in opposition to the North Sea'? I think the latter is more plausible—see [here](https://en.wikipedia.org/wiki/North_Sea_(disambiguation)):

> from the 16th century, Europeans visiting the region distinguished the "South Sea" (the Pacific Ocean south of the isthmus of Panama) from the "North Sea" (the Caribbean Sea north of the same isthmus).

The latter reading suggests not fixed but rather xcomp(identified, opposed), obl(opposed, North Sea) (cf. "Louise struck me as a fool" [here](https://universaldependencies.org/u/dep/xcomp.html), though a closer analogy would be "Louise struck me as consumed with rage").

Reply:
agreed for sure on the 2nd reading

Reply:
so then the `as` would be `case(identified, as)`?  `to` would be `mark(North Sea, to)`?

Reply:
pushed a change which might cover this (or might be horribly wrong).  is it standard to leave comments on the sentences discussing annotation corner cases?  this seems like a candidate for leaving one regarding `as opposed to` not being fixed

Reply:
> so then the `as` would be `case(identified, as)`? `to` would be `mark(North Sea, to)`?

The other way around: `mark` is for SCONJ, `case` for ADP. I'd say this "as" is SCONJ.

> pushed a change which might cover this (or might be horribly wrong). is it standard to leave comments on the sentences discussing annotation corner cases? this seems like a candidate for leaving one regarding `as opposed to` not being fixed

Not standard but I don't see any harm if you want to add one

Reply:
Does the current change look better?